src/handlers/notebook.ts

```diff
@@ -46,30 +47,36 @@ export class NotebookHandler implements IDisposable {
     }
     this.isDisposed = true;
     this.cellManager.dispose();
-    this.notebookTracker.activeCellChanged.disconnect(
-      this.onActiveCellChanged,
-      this
-    );
+    // this.notebookTracker.activeCellChanged.disconnect(
+    //   this.onActiveCellChanged,
+    //   this
+    // );
```

Comment:
I'm considering to remove this cause we have Signal.clear which should remove all signals in that instance.